rfcs/text/0000-data-citation-plan.md

```diff
@@ -0,0 +1,148 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# HCA DCP Data Citation Plan
+
+## Summary
+
+Data Contributors need to be able to cite data sets that they have contributed to the DCP.
+Data Consumers need to be able to cite the DCP data that they have used in their research projects.
+Providing citable records for the DCP will enable all contributors and consumers to reference the data stored in the DCP that they have used in their scientific publications.
+
+## Author(s)
+
+ [Trevor Heathorn](mailto:theathor@ucsc.edu)
+
+## Shepherd
+
+ [Trevor Heathorn](mailto:theathor@ucsc.edu)
+
+## Motivation
+
+There is currently no clear and agreed upon definition of the requirements for Data Citation in the DCP.
+Key issues that this RFC seeks to resolve are:
+  - What is the minimum feature set required for a first release (Phase 1)?
+  - What are the discrete set of features that make sense for subsequent releases (e.g. Phase 2, Phase 3, etc.)
+  - In which phase, if any, is support for a formal digital object identifier ([DOI](https://en.wikipedia.org/wiki/Digital_object_identifier)) required?
+
+### User Stories
+
+1. As a data contributor (e.g. researcher with a pipette), it is essential to have a unique way for my project (consisting of primary and secondary analysis data files and associated metadata files) to be identified so that others can properly cite my work when using my data, and that this citation identifier is available in the DCP Data Portal. Note: This will discourage but not prevent others from using my data without attribution.
+2. As a data consumer (e.g. researcher with a keyboard), I want to be able to view and share a unique citation identifier so that a reader of my manuscript can obtain the data needed to reproduce my results. Anyone can use the citation identifier to view and download all the original cited data and metadata files for a project from the DCP.
+3. As a data consumer, I need a simple way to reference a project in the DCP so that I can fulfill the requirements of a Creative Commons attribution license (CC-BY).
+4. As a data contributor or consumer, I need a way to use the citation identifier to access the output produced by the DCP Matrix Service for the data being cited.
+5. As a data contributor or consumer, I want to be able to update my project and provide versioned views of the data being cited.
+
+## Scientific "guardrails" [optional]
+
+*Describe recommended or mandated review from HCA Science governance to ensure that the RFC addresses the needs of the scientific community.*
+
+## Detailed Design
+
+It is proposed to split the initial implementation into three phases:
+
```

Comment:
The phase description are written assuming someone already can see what the main goal of the phase is from reading the above text. For understanability it would be nice to give the three phases tldr style titles or summary sentences for the main goal otherwise it leads to a lot of flitting backwards and forwards to understand what is being references

If I have understood correctly I would suggest

Phase 1 - stable non versioned citable URLs
Phase 2 - versioned urls and support for data release citation
Phase 3 - Enabling citation of user defined data collections

Reply:
I am correct in that phase 3 would be the fast-path for a producer who wants to cite their own data?

Reply:
Added summary titles.
Phase 3 would enable producers and consumers to cite an arbitrary set of versioned data.